packages/web-components/src/components/dotcom-shell/__stories__/dotcom-shell.stories.ts

```diff
@@ -97,10 +98,12 @@ export const Default = ({ parameters }) => {
             lang-display="${ifNonNull(langDisplay)}"
             footer-size="${ifNonNull(footerSize)}"
             user-status="${ifNonNull(userStatus)}"
+            .authenticatedProfileItems="${ifNonNull(authenticatedProfileItems)}"
```

Comment:
@asudoh just checking, is this a mock from profile-items?

Reply:
@jeffchew Yes this is a mock data.

Reply:
Thanks for confirming! 

I think we will need to eventually set up a test environment that uses live data, and we know that currently the codesandbox examples don't really cut it. I'll see if I can set up something similar to what we have for the NextJS test environments.

Also, we don't use mock data for the React version, wondering if we should be considering always using live data for our storybook instances, and only use mock data for our percy visual regression snapshots? Unless I'm misunderstanding the implementation here.

Reply:
Oh unless there is `&mock` URL query arg in place the data comes from live data.

Reply:
Thanks for confirming!